src/components/themed/TransactionListRow.tsx

```diff
@@ -189,7 +189,7 @@ export function TransactionListRow(props: Props) {
   })
 
   return (
-    <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress}>
+    <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress} marginRem={[0.5, 1]}>
```

Comment:
I thought we wanted .5 rem around all components and then a 0.5 padding around the outer-most container component on a scene?

Reply:
Yes, ideally, but we're hacking around the fact that SceneHeader was never updated to follow these rules.

Reply:
performance.measure('myMeasure', 'myMark')

Reply:
Can you clarify what you mean when you refer to the SceneHeader component?